client/src/app/components/client-gridview/client-gridview.component.ts

```diff
@@ -16,6 +16,18 @@ export class ClientGridviewComponent implements OnInit {
   dataSource: MatTableDataSource<User>;
```

Comment:
The imports are not shown here but the AfterViewInit import is not being used so please delete it.